File: integration-tests/gradle/src/main/resources/jandex-included-build-kordamp/src/main/java/org/example/ExampleResource.java
package org.example;

import jakarta.inject.Inject;
import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.Produces;
import jakarta.ws.rs.core.MediaType;
import org.acme.nested.SimpleService;

@Path("/hello")
public class ExampleResource {

    @Inject
    SimpleService simpleService;

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    public String hello() {
        return "hello " + simpleService.hello();
    }

}

File: integration-tests/gradle/src/main/resources/multi-source-project/src/main/kotlin/org/acme/ExampleResource.kt
package org.acme

import jakarta.inject.Inject
import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import jakarta.ws.rs.Produces
import jakarta.ws.rs.core.MediaType

import org.acme.service.SimpleService

@Path("/hello")
class ExampleResource {

    @Inject
    lateinit var service: SimpleService

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    fun hello(): String = service.hello()
}